### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Bazzy09**
- In this pr, i declared variable for the DOM elements.
- i also write a variable declaration that will initialize list, selectedListId and the modalOpen
- i added a function render and save that will update and save data in the local storage.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+const closeButton = document.querySelector(".close");
+const addButton = document.querySelector(".add-btn");
+const hamburger = document.querySelector(".hamburger");
+
+let lists = JSON.parse(localStorage.getItem("task.lists")) || [];
+let selectedListId = localStorage.getItem("task.selectedListId");
+let modalOpen = false;
+
+function renderAndSave() {
+  render();
```
Comment (antoHero) [inline src/index.js:27]: You are calling `render()` but where was it defined?

**Review by antoHero — CHANGES_REQUESTED** [review]
Take a look at the comment(s) and implement the requested changes.

File: src/index.js (hunk 1 above)
Comment (Bazzy09) [reply src/index.js:27]: it is a work in progress sir

Comment (antoHero) [reply src/index.js:27]: Why am I having the impression that you are seeing the solution somewhere cos I what world do you use a function before defining it?

**Pull request merged**